### PR TITLE
Update layer config and overview doc

### LINKED ISSUE #1560: Improve documentation in GitHub
Following internal discussion and reviewing repo, I believe we're deficient in documentation in the GitHub repo itself to give into on using the validation layers apart from the SDK and a basic background of the layer functionality.

If I put on my new user hat, I come to the main page in GitHub and then find ["How to Build and Run"](https://github.com/KhronosGroup/Vulkan-ValidationLayers#how-to-build-and-run) section in main readme.

When I click on that if I choose to bypass the [Vulkan Configurator Tool](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/LAYER_CONFIGURATION.md#vulkan-configurator-tool) then it immediately jumps into Activating layers on various platforms with no real info on what the validation layers are.

I'm thinking it would be nice to have a **Validation Layer Overview** section after Vulkan Configurator Tool and before the Activating Layers on Windows section. That section should explain that all Vulkan validation checks are unified into the single VK\_LAYER\_KHRONOS\_validation layer. It doesn't need to get into specifics, but can reference the [docs](https://github.com/KhronosGroup/Vulkan-ValidationLayers/tree/master/docs) dir for more info.

The other info that was brought to my attention and I'd like input on is potentially including a little historical background on the layers. This would primarily be for the benefit of people coming into Vulkan development who may be updating existing code with VK\_LAYER\_LUNARG\_standard\_validation and/or explicit individual layers. A few sentences could explain the history of those pieces and that they're now deprecated and all validation should be merged into the unified VK\_LAYER\_KHRONOS\_validation layer.

I was originally against this idea as it brings up deprecated items, but I do agree that it could be useful for devs dealing in older code who have no idea what's going on with those old layers.

### PR DESCRIPTION
**Pull request by mark-lunarg**
Added some historical context.

Fixes #1560.

### CODE REVIEW

File: LAYER_CONFIGURATION.md [hunk 1]
```diff
@@ -21,6 +21,17 @@ Vulkan validation and utility layers give Vulkan application developers the abil
...
 Developers can use the Vulkan Configurator, `vkconfig`, to enable explicit layers and disable implicit layers as well as change layer settings through a graphical user interface.  Please see [the Vulkan Configurator documentation in the Vulkan SDK](https://vulkan.lunarg.com/doc/sdk/latest/windows/vkconfig.html) for more information on using the Vulkan Configurator. Continue reading for instructions on how to configure layers without using vkconfig.
 
+## Validation Layer Overview
+
+Application validation in Vulkan is implemented solely by the [Khronos validation layer](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/docs/khronos_validation_layer.md),
+ **`VK_LAYER_KHRONOS_validation`**.
+
+Initially, validation was implemented as a dozen or so individual layers with limited scope -- i.e., `memory_tracker`, `object_tracker`, `device_limits`,
+`unique_objects`, etc., and a standard validation meta-layer, `VK_LAYER_LUNARG_standard_validation`, was supplied to enforce a particular layer order.
+In order to simplify configuration, improve build times, and standardize layer interfaces, validation was
```
Comment (sfricke-samsung) [inline LAYER_CONFIGURATION.md]: I would suggest adding: "[for more details see white paper](https://www.lunarg.com/wp-content/uploads/2019/04/UberLayer_V3.pdf)" somewhere here

Comment (mark-lunarg) [reply LAYER_CONFIGURATION.md]: Excellent suggestion, done!

**Comment by mark-lunarg** [comment]
@hevrard, will put something together shortly.

**Comment by mark-lunarg** [comment]
What else?

**Pull request merged**